odxtools/database.py

```diff
@@ -145,3 +150,7 @@ def diag_layer_containers(self, value: NamedItemList[DiagLayerContainer]) -> Non
     @property
     def comparam_subsets(self) -> NamedItemList[ComparamSubset]:
         return self._comparam_subsets
+
+    @property
+    def comparam_specs(self):
```

Comment:
```suggestion
    def comparam_specs(self) -> ComparamSpec:
```

Reply:
(`mypy` is intentionally set to "jerk mode" because it catches quite a few bugs before they bite and also, it makes it much easier to see what a function does if one knows the expected types of the arguments and their return value...)